add-on/chrome/test/mochitest/browser_LoopSidebar.js

```diff
@@ -0,0 +1,20 @@
+/* Any copyright is dedicated to the Public Domain.
+   http://creativecommons.org/publicdomain/zero/1.0/ */
+
+/*
+ * This file contains tests for the LoopSidebar
+ */
+"use strict";
+
+const { LoopSidebar } = Cu.import("chrome://loop/content/modules/LoopSidebar.jsm", {});
+
+add_task(function* test_LoopSidebar_noSidebar() {
+  let sidebar = window.document.getElementById("loop-side-browser");
+  Assert.ok(!sidebar, "there should be no sidebars yet");
+});
+
+add_task(function* test_LoopSidebar_createSidebar() {
+  LoopSidebar.createSidebar(window);
+  let sidebar = window.document.getElementById("loop-side-browser");
+  Assert.ok(sidebar, "there should create a sidebar");
```

Comment:
I forgot to mention, when loading the sidebar, we should go into offline mode to avoid any potential network activity (the tests will complain), `loadLoopPanel` in head.js has an example of how to do this.